src/soundfile.js

```diff
@@ -886,6 +886,14 @@ class SoundFile {
   }
 
   // TO DO: document this
+  /**
+   *  Set the pitch rate of a sound file. Will change the pitch and the speed also.
```

Comment:
Thanks for noting that it changes the pitch by also changing the speed, that's good for users to know! We should probably note that it assumes the default pitch is 60 (middle C), and then adjusts the pitch from there to the note you provide. e.g.:
- if you call `setPitch(61)` (C#/Db), it will raise the pitch of your sound file by 1 semitone
- if you call `setPitch(59)` (B), it will lower the pitch of your sound file by 1 semitone
- if you call `setPitch(72)`, (C5, one octave up from middle C), it will raise the pitch by one octave and double the speed
- if you call `setPitch(60)`, (C), since we assume the sound file starts at middle C, this brings it back to the original pitch and speed

Reply:
should I also add this?

Reply:
We probably don't need all those examples. We should definitely mention that the assumed input note is 60 (middle C), and then use your sentence about how the frequency and speed increases from that. Maybe it's useful saying that note 72 raises the pitch by one octave and doubles the speed to provide one concrete example.